src/client/Microsoft.Identity.Client.Broker/WamAdapters.cs

```diff
@@ -125,6 +125,28 @@ private enum ResponseStatus
             }
         }
 
+        /// <summary>
+        /// Create WAM SignOutResult Error Response
+        /// </summary>
+        /// <param name="signoutResult"></param>
+        /// <param name="logger"></param>
+        /// <exception cref="MsalServiceException"></exception>
+        internal static void ThrowExceptionFromWamError(
+            NativeInterop.SignOutResult signoutResult,
+            ILoggerAdapter logger)
+        {
+            string internalErrorCode = signoutResult.Error.Tag.ToString(CultureInfo.InvariantCulture);
+            long errorCode = signoutResult.Error.ErrorCode;
+            string errorMessage;
+
+            logger.Info("[WamBroker] Processing WAM exception");
```

Comment:
nit: verbose, since if Info or below level is already enabled, we'll know error happened was thrown from the exception itself.